controllers/imagecollector/imagecollector_controller.go

```diff
@@ -252,6 +263,7 @@ func (r *Reconciler) createImageJob(ctx context.Context, req ctrl.Request, argsC
 								},
 							},
 							SecurityContext: utils.SharedSecurityContext,
+							Env:             []corev1.EnvVar{{Name: "OTEL_EXPORTER_OTLP_ENDPOINT", Value: os.Getenv("OTEL_EXPORTER_OTLP_ENDPOINT")}, {Name: "NODE_NAME", ValueFrom: &corev1.EnvVarSource{FieldRef: &corev1.ObjectFieldSelector{FieldPath: "spec.nodeName"}}}},
```

Comment:
do we need this to be a flag?

Reply:
I think it would be set up with otlp as an env var https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md

Reply:
misunderstood this, just changed to to a flag!